api/api/V1/Reprocess.ts

```diff
@@ -38,12 +38,50 @@ const { uniq: dedupe } = lodash;
 export default (app: Application, baseUrl: string) => {
   const apiUrl = `${baseUrl}/reprocess`;
 
+  /**
+   * @api {get} /api/v1/reprocess/retryFailed/:id Retry processing a single recording which is in a failed state
+   * @apiName Reprocess
+   * @apiGroup Recordings
+   * @apiParam {Integer} id of recording to retry
+   * @apiDescription Retries processing a recording thats in a failed state
+   *
+   * @apiUse V1UserAuthorizationHeader
+   *
+   * @apiUse V1ResponseSuccess
+   * @apiUse V1ResponseError
+   */
+  app.get(
+    `${apiUrl}/retryFailed/:id`,
```

Comment:
For url endpoints, can we stick to `kebab-case` for aesthetic and consistency reasons?